backend/app.py

```diff
@@ -0,0 +1,103 @@
+from pymongo import MongoClient
+import intrinio_sdk as intrinio
+from intrinio_sdk.rest import ApiException
+from dotenv import load_dotenv
+import os
+import pandas as pd
+import yfinance
+import json
+
+from flask import Flask, jsonify, request, Response, send_from_directory
+from flask_cors import CORS, cross_origin
+
+load_dotenv()
+
+intrinio.ApiClient().set_api_key(os.environ.get("INTRINIO_KEY"))
+intrinio.ApiClient().allow_retries(True)
+
+db_client = MongoClient(os.environ.get("MONGO_URI"))
+holdings_col = db_client["Holdings"]["Holdings"]
+
+app = Flask(__name__, static_folder='build')
+CORS(app, resources={r"/*": {"origins": "*"}})
+app.config['CORS_HEADERS'] = 'Content-Type'
+
+@app.route("/refresh_cached_data/", methods=["GET"])
+@cross_origin()
+def refresh_cached_data():
+    try:
+        tickers = []
+        company_names = []
+        current_quantity = []
+        security_type = []
+        cost_basis = []
+        realtime_price = []
+        pnl = []
+        market_value = []
+        return_30D = []
+        return_1Y = []
+
+        ticker_cursor = holdings_col.find({"Cost Basis": {"$exists": True}})
+        for ticker_document in ticker_cursor:
+            print(ticker_document["ticker"])
+            if (ticker_document["active"] and ticker_document["ticker"] not in ["LVMUY", "MSBHF", "NTDOY", "VFSTX", "QQQ", "Cash & Cash Investments", "Account Total"]):
```

Comment:
What is "QQQ", "Cash & Cash Investments"?

Reply:
Also, why are we excluding some of the stocks?

Reply:
- By my understanding "Cash & Cash Investments" accounts for cash in our portfolio that is not currently being invested, and is a category on our holdings but is not something we can measure given a certain ticker as we are doing here. Because this cash is just a percentage of our portfolio that is set aside for potential investments, we wouldn't include that or count it as a stock. According to Schwab, "Cash & Cash Investments is the amount of cash available to withdraw from the account".
- The reasoning behind "excluding" certain stocks is that Intrinio simply doesn't have an equivalent ticker symbol for this select few stocks. These stocks share a commonality of being US-publicly traded stocks but all come from companies that are NOT US-based. As stated in our Intrinio Feedback Doc, there is a solution to this, which would be alternatively using the CUSIP or ISIN codes to identify that stock. However, given our current account tier with our Intrinio keys, we don't have access to doing so.
- "QQQ" should just be another ticker symbol that Intrinio doesn't have an equivalent for. @Armaanaggwl can fact-check me on that though.